src/main/java/frc/robot/Constants.java

```diff
@@ -285,7 +285,8 @@ public static final class Drive {
   public static final class EncoderInitializeConstants {
     public static final double initializedRotationsFlag = 4322.0; // must be a very high number
     public static final double initializedRotationsTolerance = 5.0;
-    public static final double absEncoderMaxZeroingThreshold = 0.95;
+    public static final double absEncoderAlmostZeroThreshold = 0.95; // intake use only
```

Comment:
this says intake use only, but is later used for the outtake to zero any wraparound which is probably not what we want, since we base the motor encoder on this value

Reply:
this results in a .05 rotation zone where we will be outside the bounds of where we should allow outtake zeroing if i understand correctly